sfa_dash/templates/forms/probabilistic_report_form.html

```diff
@@ -0,0 +1,79 @@
+{% import "forms/form_macros.jinja" as form %}
+{% extends "dash/data.html" %}
+{% block content %}
+<script src="/static/js/probabilistic-report-handling.js"></script>
+{{ metadata | safe }}
+{% include "sections/notifications.html" %}
+<h3>Create New Report</h3>
+{% include "forms/report_menu.html" %}
+<p>This is the probabilistic report mockup page</p>
+<form action="{{ url_for('forms.create_report') }}" method="post" id="report-form">
+  <div class="form-group">
+      {% if form_data is not defined %} {% set form_data = {} %}{% endif %}
+     <div class="form-element full-width">
+       {{ form.input('Name', 'text', 'name', 'Name of the Report.',
+                     required=True, form_data=form_data) }}
+     </div>
+	 <div class="form-element">
+       <label for="start">Start (UTC):</label><br/>
+       <div class="input-wrapper">
+         <input type="text" name="period-start" class="form-control" placeholder="YYYY-MM-DDTHH:MMZ" required {% if form_data %}value="{{form_data['report_parameters']['start']}}"{% endif %}></input>
+       </div>
+       {{ form.help_button('period-start') }}
+       <span class="period-start-help-text form-text text-muted help-text collapse" aria-hidden>
+           The start of the analysis period as an ISO 8601 datetime.
+       </span>
+     </div>
+     <div class="form-element">
+       <label for="end">End (UTC)</label><br/>
+       <div class="input-wrapper">
+         <input type="text" name="period-end" class="form-control" placeholder="YYYY-MM-DDTHH:MMZ" required {% if form_data %}value="{{form_data['report_parameters']['end']}}"{% endif %}></input>
+       </div>
+       {{ form.help_button('period-end') }}
+       <span class="period-end-help-text form-text text-muted help-text collapse" aria-hidden>
+           The end of the analysis period as an ISO 8601 datetime.
+       </span>
+     </div>
+     {# The following empty list is a target for the javascript found in report-handling.js
+        list items are added dynamically on page load #}
+     <h5>Observation, Probabilistic Forecast Pairs</h5>
+     <p>For each Observation, Probabilistic forecast pair selected
+        <a href="https://solarforecastarbiter.org/metrics/#crps">CRPS</a> will
+        be calculated.</p>
```

Comment:
is this old text? I thought it should be something like "For each Obsevation, Forecast pair selected, the appropriate metrics will be calculated"

Reply:
Also wondering about the name **Select Constant Values** - seems out of date with current intention (as I understood it)

Reply:
Yeah. There is a better label and explanation for the set of options that now includes the full cdf_forecast_group and its constant values, though I'm not coming up with anything quickly.
 I'm also just realizing we should probably relabel all of these **Forecast evaluation pairs** to be inline with the original Data Model document.